ludwig/utils/dataset_utils.py

```diff
@@ -3,11 +3,13 @@
 import pandas as pd
 from sklearn.model_selection import train_test_split
 
+from ludwig.api_annotations import PublicAPI
 from ludwig.constants import TEST_SPLIT, TRAIN_SPLIT, VALIDATION_SPLIT
 from ludwig.data.dataset.base import Dataset
 from ludwig.utils.defaults import default_random_seed
 
 
+@PublicAPI
```

Comment:
@dantreiman Should this be DeveloperAPI since we don't expose this directly?

Reply:
I made it @PublicAPI because it is currently referenced in the Ludwig AutoML user guide here:

<img width="793" alt="Screen Shot 2022-10-25 at 1 55 13 PM" src="https://user-images.githubusercontent.com/687280/197880134-533376f1-9741-4e62-bc70-6450fec437ef.png">

Reply:
Got it!